### ISSUE TITLE
The Hub Portal

### ISSUE THREAD

**Issue by karegapauline** [open-life-science/ols-2#19]
Project Lead: karegapauline

Mentor: lpantano and sgibson91

Welcome to OLS-2! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Life Science program :tada:.

***

Week 1 (31 August - 4 September 2020): Meet your mentor!
- [ ] Create an account on [GitHub](https://github.com)
- [ ] Check if you have access to the HackMD notes set up for your meetings with your mentor
- [ ] Prepare to meet your mentor(s) by completing a short homework provided in the HackMD notes
- [ ] Complete **your own copy** of the [open leadership self-assessment](https://docs.google.com/document/d/1oQgdfj4lPnypAyb9_Ba0Zt7E8J5L6qMvuKwu0wgQsjs/edit?usp=sharing) and share it to your mentor 
  If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
- [ ] Make sure you know when and how you'll be meeting with your mentor.

Before Week 2 (7 - 11 September 2020): Cohort Call (Welcome to Open Life Science!)
- [ ] Create an issue on the [OLS-2 GitHub repository](https://github.com/open-life-science/ols-2/issues/new) for your OLS work and share the link to your mentor.
- [ ] Draft a brief vision statement using your goals

  [This lesson](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/) from the Open Leadership Training Series (OLTS) might be helpful

- [ ] Leave a comment on this issue with your draft vision statement & be ready to share this on the call
- [ ] Check the [Syllabus](https://openlifesci.org/ols-2) for notes and connection info for all the cohort calls.

Before Week 3 (14 - 18 September 2020): Meet your mentor!
- [ ] Look up two other projects and comment on their issues with feedback on their vision statement
- [ ] Complete this [compare and contrast assignment](https://docs.google.com/document/d/1ukvqDRIYfvCapVMdE5hWP-0MkLNJ9T65X43O7F336Ac/edit?usp=sharing) about current and desired community interactions and value exchanges
- [ ] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://docs.google.com/presentation/d/1MeJo0TyuMg_waLk1J4q9y1aAqKNMuRBlnmxEChSz-cQ/edit?usp=sharing))
- [ ] Share a link to your Open Canvas in your GitHub issue
- [ ] Start your [Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/)
- [ ] Comment on your issue with your draft Roadmap
- [ ] Suggest a cohort name at the bottom of the shared notes and vote on your favorite with a +1

Before Week 4 (21 - 25 September 2020): Cohort Call (Tooling and roadmapping for Open projects)
- [ ] Look up two other projects and comment on their issues with feedback on their open canvas.

Week 5 and later
- [ ] Create a GitHub repository for your project
- [ ] Add the link to your repository in your issue
- [ ] Use your canvas to [start writing a `README.md` file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project
- [ ] Link to your README in a comment on this issue
- [ ] Add an [open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) to your repository as a file called `LICENSE.md`
- [ ] Add a [Code of Conduct](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/) to your repository as a file called `CODE_OF_CONDUCT.md`
- [ ] Invite new contributors to into your work!

This issue is here to help you keep track of work as you start Open Life Science program. Please refer to the [OLS-2 Syllabus](https://openlifesci.org/ols-2) for more detailed weekly notes and assignments past week 4.

**Comment by karegapauline** [comment]
VISION 
I'm working with science clubs in universities to create a platform that brings together young scientists from different parts of Kenya and gives them tools, resources and mentorship so that they can access local data easily and create solutions that work for our country.

**Comment by karegapauline** [comment]
Open Canvas link
https://docs.google.com/presentation/d/1-x1ARHw_oL4rvZFy-nSc2hK2lK4Rz_QVSPoGYAHXm3k/edit#slide=id.p

**Comment by crangelsmith** [comment]
Hi @karegapauline, I like the sound of your project, I have worked in some similar ideas for latin-american students (I'm happy to talk about my experience if you would like to :) ). Do you plan to focus on a particular field or make it broad? 

I tried to see the open canvas but it seems the link is not open and I need permission to access it.

**Comment by karegapauline** [comment]
Hi @crangelsmith. Sorry about the open canvas. I've fixed it. 
I would very much love to hear your experiences with your project, that would be amazing. I will focus on life sciences, particularly people headed towards bioinformatics.

**Comment by evaherbst** [comment]
Hi Karega! I really liked hearing a bit about your project during the breakout session last call and wanted to look at your open canvas. It says I don't have access to view it so I tried to request access.

**Comment by sgibson91** [comment]
@karegapauline It might be a good idea to transfer your Open Canvas to an open-by-default platform, such as hackmd or the README in your Hub Portal GitHub repo so that others can interact with it without having to request access every time?

**Comment by evaherbst** [comment]
Thanks @karegapauline for giving me access! I think you could also go in the google doc settings and change the sharing options so that anyone with a link can view.

Your Open Canvas looks great! The only recommendation I have is maybe think about how you will facilitate the interactions between students? Will this be on Github? Slack might also be a useful option!

**Comment by karegapauline** [comment]
Sorry @evaherbst and @sgibson9. I've changed permissions again. I'll put it up in hackmd as well. 

@evaherbst, I would love to have a chat about the project. This would be amazing

**Comment by evaherbst** [comment]
@karegapauline yes, let's chat about this more! Our project is similar in that we we will be using github repo and github pages to host learning materials but also want some sort of interaction, so we were thinking maybe making a Slack group for that

**Comment by karegapauline** [comment]
@evaherbst this is nice. a slack channel would definitely work

**Comment by malvikasharan** [comment]
@all-contributors please add @karegapauline for content and idea.
